data/data/powervs/variables-powervs.tf

```diff
@@ -116,6 +111,17 @@ variable "powervs_ssh_key" {
   default     = ""
 }
 
+variable "powervs_image_bucket_file_name" {
+  type        = string
+  description = "File name of the image in the COS bucket."
```

Comment:
until we have the rhcos bits from cory, what about setting the default to the current ova, and it'll then be easily change if we need to update it before that PR lands?

Reply:
That sounds good to me, changing now